src/main/java/org/sonar/report/pdf/builder/MeasuresBuilder.java

```diff
@@ -135,7 +135,11 @@ private void addMeasures(final Measures measures,
     query.setDepth(0);
     query.setIncludeTrends(true);
     Resource resource = sonar.find(query);
-    this.addAllMeasuresFromDocument(measures, resource);
+    if (resource != null) {
+      this.addAllMeasuresFromDocument(measures, resource);
+    } else {
+      LOG.warn("Empty response when looking for measures: " + measuresAsString.toString());
```

Comment:
I guess an empty response is valid, otherwise we would get an exception. In that case, yeah, DEBUG is a better choice. I just updated this with another commit.

And thank *you* for a very useful plugin :)